src/main/java/se/sundsvall/messaging/api/StatusAndHistoryResource.java

```diff
@@ -172,7 +158,26 @@ ResponseEntity<List<DepartmentStatistics>> getDepartmentStatistics(
 		@RequestParam(name = "from", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @Parameter(description = "From-date (inclusive). Format: yyyy-MM-dd (ISO8601)") final LocalDate from,
 		@RequestParam(name = "to", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @Parameter(description = "To-date (inclusive). Format: yyyy-MM-dd (ISO8601)") final LocalDate to) {
 
-		return ResponseEntity.ok(statisticsService.getDepartmentLetterStatistics(origin, department, from, to, municipalityId));
+		return ok(statisticsService.getDepartmentLetterStatistics(origin, department, from, to, municipalityId));
+	}
+
+	@Operation(summary = "Get messages sent by a user")
+	@GetMapping(value = USER_MESSAGES_PATH, produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
+	ResponseEntity<UserMessages> getUserMessages(final @Valid UserMessagesRequest request,
```

Comment:
See comment in UserMessagesRequest class

Reply:
![Uploading image.png…]()